fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -2977,6 +3001,21 @@ public boolean isEnableFillDataCache(Partition partition) {
         }
     }
 
+    // Read indexes and assign indexId for some compatible reasons when upgrade from old version
+    private void tryToAssignIndexId() {
+        if (this.indexes != null && !this.indexes.getIndexes().isEmpty()) {
```

Comment:
Will this index id only be used for inverted indexes? Why don't we consider the impact of bitmap index on the maximum index id?

Reply:
It's more complicated when involve the compatibility and consistence of index meta between BE and FE when upgrade cluster from the old, So I want this pr only to make a change as little as possible to existing index type.